README.md

```diff
@@ -102,9 +106,9 @@ To update the state of your currently deployed integrations, hover over the **Ap
 
 ![Apache Camel K Integrations view - Refresh](images/camelk-integrations-view-refresh-action.jpg)
 
-## Starting a new Camel K Integration with multiple parameters
+## Creating a new Camel K Integration with multiple parameters
 
-Using the right-click menu is the easiest way to start a new integration and works well for simple cases. When the Camel K integration requires more configuration, you can set that up using a Task.
+The right-click menu is providing by default several ways to start a new integration and works well for simple cases. When the Camel K integration requires more configuration, you can set that up using a Task.
```

Comment:
Might change this to:

Though the simple "Start Apache Camel K Integration" menu works well for simple cases, you can use a Task for more complex integrations.